### PR TITLE
Elias/issue#7/menu styling when onpage

### LINKED ISSUE #7: Navigation items should have some kind of styling for a given link when on that link's page
Each page shares a copy/pasted navigation area.

For each navigation area, when on a given page, for example clients.html, the clients link in the navbar should have some special styling that indicates that it is the active page.

Follow design principles, and color pallete, linked at top of css file. 

Maybe some kind of underline? Up to you.

### PR DESCRIPTION
**Pull request by Figgy24-20**
issue #7 

Nav items have new class="on-page" which styles the links blue when on a given page. 
on-page also styles them to have a dark red #1e0d0a shadow

### CODE REVIEW

File: src/css/style.css [hunk 1]
```diff
@@ -238,6 +238,11 @@ nav li a:visited {
   margin-bottom: 5px;
 }
 
+nav li a.on-page {
+  color: #92dce5;
```
Comment (komali2) [inline src/css/style.css:242]: ideally colors are always variables, at line `8` see the variables defined at `:root`. This also means all colors will be within the palette we chose a bit back.

Comment (komali2) [reply src/css/style.css:242]: Thus yeah this color isn't in the palette so I feel like it doesn't quite match:

File: src/css/style.css [hunk 2]
```diff
@@ -238,6 +238,11 @@ nav li a:visited {
   margin-bottom: 5px;
 }
 
+nav li a.on-page {
+  color: #92dce5;  
+  text-shadow: 2px 2px 3px #1e0d0a;
```
Comment (komali2) [inline src/css/style.css:243]: as for the text shadow, kinda nice, but again same comment regarding color that isn't a predefined variable. Also, I find this drop shadow makes it a bit difficult to read, the top left of all the letters are like, white or something, idk, but it's not very contrasty

A pure color change alone is probably enough to accomplish our goal of letting the user know which page they're on. our page is really flat with no other places that have drop shadows. If we do use them, i think it makes more stylistic sense to start using them on concepts like Cards (see https://m3.material.io/)

Oh, or perhaps changing the underline type, plus the color, bearing in mind that for colorblind users a color change alone isn't quite enough

File: src/engineering.html [hunk 3]
```diff
@@ -39,13 +39,13 @@ <h2 class="mobile">
           <ol>
             <li><a href="index.html">Home</a></li>
             <li><a href="clients.html">Client</a></li>
-            <li><a href="engineering.html">Engineer</a></li>
+            <li><a href="engineering.html" class="on-page" >Engineer</a></li>
```
Comment (komali2) [inline src/engineering.html:42]: this strategy is perfectly acceptable and how i would have done it

**Review by komali2 — CHANGES_REQUESTED** [review]

File: src/css/style.css (hunk 2 above)
Comment (Figgy24-20) [reply src/css/style.css:243]: Sure I'll switch it to variables, and make a more matching color. Just to you know, the color I used is from the color palate linked at the top of the css.

**Pull request closed without merging**